### PR TITLE
Create cividis.jl

### PR DESCRIPTION
**Pull request by briochemc**
Not sure how to add colormaps like this, but I thought this one was missing, so I tried to add it 😄 

I guess it must be added to the docs too but I'm not sure where. And probably needs some tests too :)

Some pointers/tips would be much helpful! 😃

### CODE REVIEW

**Comment by cormullion** [comment]
Hi! Good idea... Will this not clash with the one in Plots.jl (http://docs.juliaplots.org/latest/generated/colorschemes/#Pre-defined-ColorSchemes)? 

Schemes are automatically documented during the build process, as long as they belong to one of the categories (https://github.com/JuliaGraphics/ColorSchemes.jl/blob/master/docs/src/basics.md#pre-defined-schemes). I'm hoping to redo these in SVG, once the next release of Documenter.jl is out.

**Comment by briochemc** [comment]
> Hi! Good idea... Will this not clash with the one in Plots.jl (http://docs.juliaplots.org/latest/generated/colorschemes/#Pre-defined-ColorSchemes)?

Oh good catch! I knew I saw/used it before... I was just surprised I could not find it directly in ColorSchemes... Sorry for the noise then?

**Comment by cormullion** [comment]
Good practice though?! 😄

**Comment by briochemc** [comment]
Could you ELI5 why these "predefined" colors in Plots were not moved to ColorSchemes? 🙏

**Comment by cormullion** [comment]
No 😄

**Comment by cormullion** [comment]
But the surgeons who performed the operation of integrating ColorSchemes.jl into Plots.jl may know.
